### PR TITLE
Using routing from http

### PR DESCRIPTION
**Pull request by g4s8**
#10 - updated http module to `0.7.1` version, using routing classes from http module

### CODE REVIEW

File: src/main/java/com/artipie/files/FilesSlice.java [hunk 1]
```diff
@@ -23,79 +23,38 @@
...
-                break;
-            default:
-                response = new RsWithStatus(RsStatus.NOT_FOUND);
-                break;
-        }
-        return response;
+        super(
+            new SliceRoute(
+                new SliceRoute.Path(new RtRule.ByMethod(RqMethod.GET), new SliceDownload(storage)),
+                new SliceRoute.Path(new RtRule.ByMethod(RqMethod.PUT), new SliceUpload(storage)),
```
Comment (Vatavuk) [inline src/main/java/com/artipie/files/FilesSlice.java]: @g4s8 I believe we are missing handling of POST request. It should be SliceUpload for POST as well.

**Comment by 0crat** [comment]
This pull request [#11](https://github.com/artipie/files-adapter/issues/11) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/7934d304-86f4-4b9a-a724-22a7929e6094); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/files/FilesSlice.java (hunk 1 above)
Comment (g4s8) [reply src/main/java/com/artipie/files/FilesSlice.java]: @Vatavuk I don't think so: `POST` request doesn't specify resource id, but we need it to upload the file. So I think it's correct to accept only `PUT` request in this case. https://restfulapi.net/rest-put-vs-post/

Comment (Vatavuk) [reply src/main/java/com/artipie/files/FilesSlice.java]: @g4s8 The previous version of `response` method supported `POST` and was returning response object just like for `PUT` that's why I commented. Perhaps that was a bug then... anyway I have no further comments so we can merge.

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @vatavuk[/z](https://www.0crat.com/u/vatavuk)[/z](https://www.0crat.com/u/vatavuk) didn't make CR comments

**Pull request merged**